configs/_base_/schedules/schedule_20k_epochwise.py

```diff
@@ -0,0 +1,9 @@
+# optimizer
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0005)
+optimizer_config = dict()
+# learning policy
+lr_config = dict(policy='poly', power=0.9, min_lr=1e-4, by_epoch=True)
+# runtime settings
+runner = dict(type='IterBasedRunner', max_iters=20000)
+checkpoint_config = dict(by_epoch=True, interval=2000)
+evaluation = dict(interval=2000, metric='mIoU')
```

Comment:
This config may not be necessary.

Reply:
ok, I will clean these codes in next commit.